packages/conf-mbedtls/conf-mbedtls.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+name: "conf-mbedtls"
+authors: "ARM Limited"
+maintainer: "Andy Li <andy@onthewings.net>"
+homepage: "https://tls.mbed.org/"
+bug-reports: "https://github.com/ARMmbed/mbedtls/issues"
+license: "Apache-2.0"
+build: [
+  ["ls" "/usr/include/mbedtls/version.h"] {os != "macos"}
+  ["brew" "ls" "--versions" "mbedtls"] {os = "macos"}
```

Comment:
I don't think this is a good solution to test whether this package is available or not.
Could you change this using a test file? For instance:
```
$ cat files/test.c
#include <mbedtls/version.h>
$ cat opam
[...]
build: ["cc" "test.c"]
```

Reply:
Thanks for the review. Just updated the PR as suggested.